.github/workflows/ui_test.yml

```
@@ -29,6 +29,8 @@ jobs:
         env:
           VITE_PROJECT_ID: ${{ secrets.VITE_DEV_PROJECT_ID }}
           VITE_EXPLORER_API_URL: ${{ secrets.VITE_EXPLORER_API_URL }}
+          TEST_DAPP_ID: ${{ secrets.TEST_DAPP_ID }}
+          TEST_DAPP_SECRET: ${{ secrets.TEST_DAPP_SECRET }}
```

In line with suggested renaming above:

```suggestion
          TEST_DAPP_PROJECT_SECRET: ${{ secrets.TEST_DAPP_PROJECT_SECRET }}
```